app/forms/idv/profile_form.rb

```diff
@@ -97,12 +95,5 @@ def parsed_dob
     rescue
       nil
     end
-
-    def result
-      {
-        success: success,
-        errors: errors.messages,
```

Comment:
They're recorded via the FormResponse that is returned by ProfileStep. We have existing `Verify::SessionsController` specs that check for analytics and are passing after this refactor.